waitress/pantry/viewsets.py

```diff
@@ -0,0 +1,97 @@
+from datetime import date
+
+import pytz
+from django.shortcuts import get_object_or_404
+from drf_yasg import openapi
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import status as status_code
+from rest_framework import viewsets
+from rest_framework.decorators import action
+from rest_framework.response import Response
+
+from app.decorators import guard
+from app.models import Pantry, SlackUser, Passphrase, MealSession
```

Comment:
The Pantry model should live inside the pantry app too.

The aim of this ticket is to ensure we properly separate concerns - since the pantry model belongs to the Pantry app then it should be moved to the `pantry/models.py`